controllers/storageclient/badgerdb/simple.go

```diff
@@ -179,234 +176,153 @@ func (cl Client) getUserCount(applicationName string, version int, signature str
 	return count, nil
 }
 
-// GetSummaryMetrics gets summarized metrics for a particular application, version, and signature
-func (cl Client) GetSummaryMetrics(applicationName string, version int, signature string) (*storageclient.VersionMetricSummary, error) {
-	count, err := cl.getUserCount(applicationName, version, signature)
-	if err != nil {
-		return nil, fmt.Errorf("cannot get user count: %e", err)
-	}
+// GetMetrics returns a nested map of the metrics data for a particular application, version, and signature
+// Example:
+//
+//	{
+//		"my-metric": {
+//			"MetricsOverTransactions": [1, 1, 3, 4, 5]
+//			"MetricsOverUsers": [2, 7, 5]
+//		}
+//	}
+//
+// NOTE: for users that have not produced any metrics (for example, via lookup()), GetMetrics() will add 0s for the extra users in metricsOverUsers
+//
+// Example, given 5 total users:
+//
+//	{
+//		"my-metric": {
+//			"MetricsOverTransactions": [1, 1, 3, 4, 5]
+//			"MetricsOverUsers": [2, 7, 5, 0, 0]
+//		}
+//	}
```

Comment:
This note might need to be documented elsewhere

Reply:
@Alan-Cha You documented this in the interface. Perhaps that is sufficient. Having it here again isn't a big deal.